src/SFA.DAS.ApplyService.Web/Infrastructure/ApplicationApiClient.cs

```diff
@@ -168,5 +171,31 @@ public async Task<GetOversightReviewResponse> GetOversightReview(Guid applicatio
         {
             return await Get<GetOversightReviewResponse>($"/Oversight/{applicationId}/review");
         }
+
+        public async Task<List<AssessorSequence>> GetClarificationSequences(Guid applicationId)
+        {
+            return await Get<List<AssessorSequence>>($"/Clarification/Applications/{applicationId}/Overview");
+        }
+
+        public async Task<List<ClarificationPageReviewOutcome>> GetAllClarificationPageReviewOutcomes(Guid applicationId, string userId)
+        {
+            return await Post<GetAllClarificationPageReviewOutcomesRequest, List<ClarificationPageReviewOutcome>>($"/Clarification/Applications/{applicationId}/GetAllPageReviewOutcomes", new GetAllClarificationPageReviewOutcomesRequest(applicationId, userId));
+        }
+
+        public async Task<AssessorPage> GetAssessorPage(Guid applicationId, int sequenceNumber, int sectionNumber, string pageId)
+        {
+            AssessorPage assessorPage;
+
+            if (string.IsNullOrEmpty(pageId))
+            {
+                assessorPage = await Get<AssessorPage>($"/Assessor/Applications/{applicationId}/Sequences/{sequenceNumber}/Sections/{sectionNumber}/Page");
+            }
+            else
+            {
+                assessorPage = await Get<AssessorPage>($"/Assessor/Applications/{applicationId}/Sequences/{sequenceNumber}/Sections/{sectionNumber}/Page/{pageId}");
+            }
+
+            return assessorPage;
+        }
```

Comment:
A new Oversight API client might be better here so that we separate off the logic

Reply:
ok sounds good